language-plutus-core/.hlint.yaml

```diff
@@ -1,27 +0,0 @@
----
```

Comment:
irrelevant deletions?

Reply:
Yes, I've fixed this with the new commits.